dev/archery/archery/cli.py

```diff
@@ -450,19 +511,57 @@ def benchmark_run(ctx, rev_or_path, src, preserve, output, cmake_extras,
     \b
     archery benchmark run --output=run.json
     """
+    if language == "cpp":
+        ctx.forward(benchmark_run_cpp, **kwargs)
+    elif language == "java":
+        ctx.forward(benchmark_run_java, **kwargs)
+
+
+@benchmark.command(name="run_cpp")
+@click.pass_context
+def benchmark_run_cpp(ctx, rev_or_path, src, preserve, output, cmake_extras,
```

Comment:
This division also seems unnecessary; CppBenchmarkRunner and JavaBenchmarkRunner seem similar enough to share more code here